src/com/itacademy/eshop/product/Card.java

```diff
@@ -0,0 +1,24 @@
+package com.itacademy.eshop.product;
+
+public class Card extends Payment {
+
+    private int cardNumber;
+    private String cardHolderName;
```

Comment:
private field is never set, should either include it in a constructor or create a setter for it